src/management/OrganizationsManager.js

```diff
@@ -0,0 +1,816 @@
+ * @memberOf  module:management.OrganizationsManager.prototype
+ *
+ * @example
+ * var params =  { id :'ORGANIZATION_ID', connection_id: 'CONNECTION_ID' };
+ *
+ * management.organizations.removeEnableConnection(params, data, function (err) {
+ *   if (err) {
+ *     // Handle error.
+ *   }
+ * });
+ *
+ * @param   {String}    params.id             ID of the Organization.
+ * @param   {String}    params.connection_id  ID of the Connection.
+ * @param   {Object}    data                  Updated connection.
+ * @param   {Function}  [cb]                  Callback function.
+ *
+ * @return  {Promise|undefined}
+ */
+
+OrganizationsManager.prototype.updateEnabledConnection = function(params, data, cb) {
+  data = data || {};
+  params = params || {};
+
+  if (!params.id) {
+    throw new ArgumentError('The organization ID passed in params cannot be null or undefined');
+  }
+  if (typeof params.id !== 'string') {
+    throw new ArgumentError('The organization ID has to be a string');
+  }
+
+  if (!params.connection_id) {
+    throw new ArgumentError('The connection ID passed in params cannot be null or undefined');
+  }
+  if (typeof params.connection_id !== 'string') {
+    throw new ArgumentError('The connection ID has to be a string');
+  }
+
+  if (cb && cb instanceof Function) {
+    return this.connections.patch(params, data, cb);
+  }
+
+  return this.connections.patch(params, data);
+};
+
+/**
+ **** Organization Members
+ */
+
+/**
+ * Get Members in a Organization
+ *
+ * @method    getMembers
+ * @memberOf  module:management.OrganizationsManager.prototype
+ *
+ * @example
+ * var params = {id : 'ORGANIZATION_ID'}
+ * @example <caption>
+ *   This method takes an organization ID and returns the members in an Organization
+ * </caption>
+ *
+ * management.organizations.getMembers( {id : 'ORGANIZATION_ID'}, function (err, members) {
+ *   console.log(members);
+ * });
+ *
+ * @param   {String}    [organization_id]   Organization ID
+ * @param   {Function}  [cb]                Callback function.
+ *
+ * @return  {Promise|undefined}
+ */
+OrganizationsManager.prototype.getMembers = function(params, callback) {
+  return this.members.getAll(params, callback);
+};
+
+/**
+ * Get a Member in a Organization
+ *
+ * @method    getMember
+ * @memberOf  module:management.OrganizationsManager.prototype
+ *
+ * @example
+ * var params = {id : 'ORGANIZATION_ID', user_id: 'USER_ID'}
+ * @example <caption>
+ *   This methods takes the organization ID and user ID and returns the member
+ * </caption>
+ *
+ * management.organizations.getMember( {id : 'ORGANIZATION_ID', user_id: 'USER_ID'}, function (err, member) {
+ *   console.log(member);
+ * });
+ *
+ * @param   {String}    [organization_id]   Organization ID
+ * @param   {Function}  [cb]                Callback function.
+ *
+ * @return  {Promise|undefined}
+ */
+OrganizationsManager.prototype.getMember = function(params, callback) {
+  return this.members.get(params, callback);
+};
```

Comment:
We don't offer a get **single** member on API2. (we have it in groups-api and we use it on server for membership test)